.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,53 @@
+name: Run test suite
+on: [push, pull_request]
+
+jobs:
+  check_version:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [ 3.8 ]
+        os: [ ubuntu-latest ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.8 ]
+        os: [ ubuntu-latest, macOS-latest, windows-latest ]
+      fail-fast: false
+    needs: [ check_version ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+
+      - name: Test with pytest
+        run: |
+          python setup.py install
```

Comment:
Ideally it would be either or both of `python setup.py sdist` / `python setup.py bdist_wheel`, and then installed from the sdist/wheel: `pip install dist/<file>`.

This helps make sure that everything that needs to go into the sdist/wheel really does end up in the sdist or wheel.